modules/huawei/CMakeLists.txt

```diff
@@ -30,7 +30,7 @@ target_link_libraries(hwivs
     ${IVS_SDK_LIBRARY}
 )
 
-target_compile_options(hwivs PRIVATE -Wall -Wextra -Wunused-parameter)
```

Comment:
Is there any problem with the `-Wextra -Wunused-parameter`?

Reply:
It throws an error while run `cmake`. I forgot the specific error messages, but I found a suggestion where he said just remove it.